pkg/transformer/openshift/openshift.go

```diff
@@ -135,6 +135,24 @@ func getAbsBuildContext(context string, composeFileDir string) (string, error) {
 		return "", err
 	}
 	prefix := strings.Trim(string(out), "\n")
+	index := strings.Index(context, prefix)
+	if index != -1 {
+		if prefix == "" {
```

Comment:
i feel that this can be split out to two if statements instead of being nested.
ex:
```
if index != -1 {
...
} else if prefix == "" {
...
}
```